.github/workflows/ci.yml

```diff
@@ -53,13 +53,21 @@ jobs:
           - arm-unknown-linux-gnueabihf # ARMv6 Linux, hardfloat (kernel 3.2, glibc 2.17)
           - aarch64-unknown-linux-gnu # ARM64 Linux (kernel 4.2, glibc 2.17+)
           - x86_64-unknown-linux-gnu # 64-bit Linux (kernel 2.6.32+, glibc 2.11+)
+          - aarch64-unknown-linux-musl # ARM64 Linux with MUSL
+          - x86_64-unknown-linux-musl # 64-bit Linux with MUSL
     runs-on: ubuntu-latest
     steps:
+      - name: Set X86_64 environment variables
+        run: export CARGO_TARGET_X86_64_UNKNOWN_LINUX_MUSL_RUSTFLAGS="-C target-feature=-crt-static"
+      - name: Set aarch64 environment variables
+        run: export CARGO_TARGET_AARCH64_UNKNOWN_LINUX_MUSL_RUSTFLAGS="-C target-feature=-crt-static"
```

Comment:
If these steps work similar to ADO pipeline tasks, this variable may not be persisted across steps. It might need to be defined under `env`

Reply:
Yeah I should have done it this way:

```
jobs:
  greeting_job:
    runs-on: ubuntu-latest
    env:
      Greeting: Hello
```